lib/milc_interface.cpp

```diff
@@ -368,7 +369,7 @@ void qudaUpdateUPhasedPipeline(int prec, double eps, QudaMILCSiteArg_t *arg, int
 
 void qudaUpdateUPhased(int prec, double eps, QudaMILCSiteArg_t *arg, int phase_in)
 {
-  qudaUpdateUPhasedPipeline(prec, eps, arg, 0, 0);
+  qudaUpdateUPhasedPipeline(prec, eps, arg, phase_in, 0);
```

Comment:
should that also go into 1.1 ?

Reply:
Yes.  I had meant to back port that, but had forgotten.

Reply:
#1175